components/camera/rtsp/rtsp_test.go

```diff
@@ -2,18 +2,22 @@ package rtsp
 
 import (
 	"context"
+	"fmt"
 	"strings"
 	"testing"
+	"time"
 
 	"github.com/bhaney/rtsp-simple-server/server"
 	"github.com/edaniels/golog"
 	ffmpeg "github.com/u2takey/ffmpeg-go"
 	"go.viam.com/test"
 	viamutils "go.viam.com/utils"
 	"go.viam.com/utils/artifact"
+
+	"go.viam.com/rdk/components/camera"
 )
 
-func startFFMPEG(outputURL string, streamStarted chan struct{}) context.CancelFunc {
```

Comment:
[q] Is the idea here that we're going to wait 10 seconds and retry to connect to the server anyways, so there's no reason to block anymore?

Reply:
that, and the blocking didn't do much anyway! Even though the FFMpeg stream had started and closed the channel, there was still a few seconds of time before the stream was actually getting published to the server.